recipes/metamdbg/meta.yaml

```diff
@@ -25,10 +25,8 @@ requirements:
   run:
     - zlib
     - openmp
-    - wfmash =0.8.2
-    - samtools =1.6
     - minimap2 =2.24
```

Comment:
Any chance minimap2 2.28 could be used instead ? This will make it possible to provide linux-aarch64 build too!

Reply:
https://github.com/bioconda/bioconda-recipes/pull/49833

Reply:
Yes there should be no problem, I have added it as an issue on the metaMDBG github.